packages/tabs/examples/default-tabs.md

```diff
@@ -33,48 +33,52 @@ const StyledSpacedField = styled(Field)`
   margin-bottom: ${props => props.theme.space.sm};
 `;
 
-<Grid>
-  <Row>
-    <Col md={4}>
-      <Well recessed>
-        <StyledSpacedField>
-          <Toggle
-            checked={state.isVertical}
-            onChange={e => setState({ isVertical: e.target.checked })}
+<State initialState={initialState}>
```

Comment:
The `initialState` global actually does some state "magic" with styleguidist. It may be cleaner to just move that object inline to this prop.

Reply:
Oh no, magic. What do you think about declaring these variables using `const` instead? That way styleguist can't re-declare another value for `initialState`. 

// before
```jsx
tabs = ['Tab 1', 'Tab 2', 'Tab 3'];

initialState = {
  isVertical: false,
  isDisabled: false,
  selectedItem: tabs[0]
};
```

// before
```jsx
const tabs = ['Tab 1', 'Tab 2', 'Tab 3'];

const initialState = {
  isVertical: false,
  isDisabled: false,
  selectedItem: tabs[0]
};
```

Reply:
👍 Looks good to me!